src/fiatExchanges/quotes/constants.ts

```diff
@@ -1,20 +1,12 @@
-import {
-  FiatAccountSchema,
-  FiatAccountSchemas,
-  SupportedOperatorEnum,
-} from '@fiatconnect/fiatconnect-types'
+import { FiatAccountSchema, FiatAccountSchemas } from '@fiatconnect/fiatconnect-types'
 
 type DefaultAllowedValues = Partial<{
   [Schema in FiatAccountSchema]: {
     [Property in keyof Partial<FiatAccountSchemas[Schema]>]: string[]
   }
 }>
 
-export const DEFAULT_ALLOWED_VALUES: DefaultAllowedValues = {
-  [FiatAccountSchema.MobileMoney]: {
-    operator: Object.keys(SupportedOperatorEnum),
-  },
-}
+export const DEFAULT_ALLOWED_VALUES: DefaultAllowedValues = {}
```

Comment:
seems like this isn't used anymore. please get rid of it.

Reply:
Removed it.